changedetectionio/tests/test_diff_filter_only_additions.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3
+
+import time
+from flask import url_for
+from .util import live_server_setup
+
+def set_original_response():
+    test_return_data = """
+        A few new lines
```

Comment:
I was thinking that more test data here could be good, like a much more test case perhaps

Reply:
are you saying adding a bunch to the single test case or adding a bunch more situations?